pages/index.vue

```diff
@@ -524,7 +524,7 @@ export default {
 
   .desc-section__part {
     width: 100%;
-    text-align: justify;
+    text-align: left;
```

Comment:
Revert this

Reply:
@1Spinne do you wan't to do this
following: https://discord.com/channels/454739001841025024/533334373026627585/879816758423355482